From 4504e47a842a5101ba7c0fca63d40bf94e09bf6a Mon Sep 17 00:00:00 2001
From: Xufeng Zhang <xufeng.zhang@windriver.com>
Date: Thu, 26 May 2011 10:40:45 +0800
Subject: [PATCH] crypto/ocf: Remove wrong dependency for OCF_TALITOS

commit 001f88c4d7940263c382b52326a7aa8a5c6ffd63 add dependency
CRYPTO_DEV_TALITOS for OCF_TALITOS, this is not true, CRYPTO_DEV_TALITOS
and OCF_TALITOS are mutually exclusive.
So remove this dependency, and also select PPC_MERGE when OCF_TALITOS
is enabled.

Signed-off-by: Xufeng Zhang <xufeng.zhang@windriver.com>
---
 crypto/ocf/Kconfig |    1 +
 1 files changed, 1 insertions(+), 0 deletions(-)

diff --git a/crypto/ocf/Kconfig b/crypto/ocf/Kconfig
index 93ecc66..96d4824 100644
--- a/crypto/ocf/Kconfig
+++ b/crypto/ocf/Kconfig
@@ -69,6 +69,7 @@ config OCF_HIFNHIPP
 config OCF_TALITOS
 	tristate "talitos (HW crypto engine)"
 	depends on OCF_OCF && CRYPTO_DEV_TALITOS
+	select PPC_MERGE
 	help
 	  OCF driver for Freescale's security engine (SEC/talitos).
 
-- 
1.6.5.2

